docs/index.md

```diff
@@ -3,7 +3,7 @@
 page_title: "nobl9 Provider"
 subcategory: ""
 description: |-
-
+  
 ---
 
 # nobl9 Provider
```

Comment:
can you please also update the version in the example to be `0.2.0`? I guess with CompositeSLOs we will release 0.2